File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/NumericUpDown/Functionality/MinimumPropertypage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.NumericUpDown.Functionality
{
    public partial class MinimumPropertypage : UserControl
    {
        public MinimumPropertypage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles the Load event of the MinimumPropertypage control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void MinimumPropertypage_Load(object sender, EventArgs e)
        {
            // Set initial value for NumericUpDown that represents 
            // minimum for the demonstrated NumericUpDown.
            this.minimumNumericUpDown.Value = this.demoNumericUpDown.Minimum;
        }

        /// <summary>
        /// Handles the ValueChanged event of the minimumNumericUpDown control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void minimumNumericUpDown_ValueChanged(object sender, EventArgs e)
        {
            this.demoNumericUpDown.Minimum = this.minimumNumericUpDown.Value;
        }

    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/NumericUpDown/Functionality/MinimumPropertypage.vb
Namespace CompanionKit.Controls.NumericUpDown.Functionality

	Public Class MinimumPropertypage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

	        

		End Sub

        ''' <summary>
        ''' Handles the ValueChanged event of the minimumNumericUpDown control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub minimumNumericUpDown_ValueChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles minimumNumericUpDown.ValueChanged
            Me.demoNumericUpDown.Minimum = Me.minimumNumericUpDown.Value
        End Sub

        ''' <summary>
        ''' Handles the Load event of the MinimumPropertypage control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub MinimumPropertypage_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
            ' Set initial value for NumericUpDown that represents 
            ' minimum for the demonstrated NumericUpDown.
            Me.minimumNumericUpDown.Value = Me.demoNumericUpDown.Minimum
        End Sub
    End Class

End Namespace